core/src/main/java/org/bitcoinj/core/Transaction.java

```diff
@@ -520,18 +522,40 @@ public boolean isEveryOwnedOutputSpent(TransactionBag transactionBag) {
 
     /**
      * Returns the earliest time at which the transaction was seen (broadcast or included into the chain),
-     * or the epoch if that information isn't available.
+     * or empty if that information isn't available.
      */
+    public Optional<Instant> getUpdateTimeInstant() {
```

Comment:
`getUpdateInstant()`?

Reply:
I don't think we've been entirely consistent on this, then -- I'm pretty sure we already have cases with just `.getXxxInstant()`.  I think you'll be able to find them well enough with `/get*Instant()` and with the shorter names it's ok if we don't make the second change right away.

Either way is fine with me, though.